app/views/shared/address_selection/_confirm_address.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= page_template(
+      page_title: t(".heading"),
+      template: :basic,
+      form:,
+    ) do %>
+
+      <h1 class="govuk-heading-xl"><%= t(".heading") %></h1>
+
+      <p class='govuk-body'>1 address found for
+        <span class="govuk-!-font-weight-bold"><%= @form.postcode.insert(-4, " ") %></span>
+        <% unless @form.model.building_number_name.empty? %>
```

Comment:
This is an unless, but the other form is `if @form.model.building_number_name.present?`
Should they be consistent?

Reply:
I think I've updated this in a subsequent commit. Should probably have done it in the same commit, but they are consistent now